server/tserver/src/main/java/org/apache/accumulo/tserver/session/DefaultSessionComparator.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.tserver.session;
+
+public class DefaultSessionComparator extends SessionComparator {
+
+  @Override
+  public int compareSession(Session sessionA, Session sessionB) {
```

Comment:
I did not carry this forward in #549 for the following reasons :
 * Its goal is not documented and its hard to understand the goal looking at the code.
 * Its seems to prioritize newer scans with an exception for scans with "large" idle times.  However, I think for two scans that have never run it should prioritize the oldest.  But this goes back to the goal not being documented.
  * I think there may be overflow bugs with casting of long to int, but maybe this was intentional?? Not sure.